packages/plugin/webpack/src/WebpackPlugin.ts

```diff
@@ -164,8 +164,8 @@ Your packaged app may be larger than expected if you dont ignore everything othe
       if (entryPoint.preload) {
         defines[`${entryPoint.name.toUpperCase().replace(/ /g, '_')}_PRELOAD_WEBPACK_ENTRY`] =
           this.isProd
-          ? `\`file://\$\{require('path').resolve(__dirname, '../renderer', '${entryPoint.name}', 'preload.js')\}\``
-          : `'${path.resolve(this.baseDir, 'renderer', entryPoint.name, 'preload.js')}'`;
+          ? `\`\$\{require('path').resolve(__dirname, '../renderer', '${entryPoint.name}', 'preload.js')\}\``
```

Comment:
Why doesn't this part of the ternary expression need the escaping logic?

Reply:
It doesn't need it, because the result is an expression. (Keep in mind, the webpackDefine plugin does not define the variable, it just replaces MAIN_WINDOW_PRELOAD_WEBPACK_ENTRY in the source with whatever string it's given)


In production
```javascript
{
  preload: MAIN_WINDOW_PRELOAD_WEBPACK_ENTRY
}
```
will get transformed to
```javascript
{
  preload: require('path').resolve(__dirname, '../renderer', 'MAIN_WINDOW', 'preload.js')
}
```


Whereas in development, the result is a string and needs to get properly escaped since
```javascript
MAIN_WINDOW_PRELOAD_WEBPACK_ENTRY = `'${path.resolve(this.baseDir, 'renderer', entryPoint.name, 'preload.js')}'`
```

results in
```javascript
{
  preload: '\basedir\renderer\main\preload.js'
}
```
However, it has to be
```javascript
{
  preload: '\\basedir\\renderer\\main\\preload.js'
}
```